resolver/src/test/java/io/netty/resolver/DefaultHostsFileEntriesResolverTest.java

```diff
@@ -28,6 +32,21 @@
     @Test
     public void testLocalhost() {
         DefaultHostsFileEntriesResolver resolver = new DefaultHostsFileEntriesResolver();
+        //make sure there's _some_ localhost entry. put one if there isnt
```

Comment:
...maybe make the map field (DefaultHostsFileEntriesResolver.entries) non-final, replace it with a Mock in this test and assert that whatever string is searched for is lowercased (which is the real functionality being tested).